chatterbot/corpus/data/english/claims.corpus.json

```diff
@@ -0,0 +1,10 @@
+{
+    "claims": [
+        "I just went to the doctor",
+        "I got a prescription for @drug, is it covered?",
+        "My doctor says I need @procedure, how much will it cost?",
+        "I broke my arm, what should I do?",
+        "I'm having @procedure down, what is the cost?",
+        "I had @procedure done, what can I do now?"
```

Comment:
It appears that these statements are unstructured. When ChatterBot reads this file it assumes that each list of statements is a conversation. For example:

> Person1: Hi
> Person2: How are you?
> Person1: I am good.
> Person2: Am am glad to hear that.
> Person1: Thank you

In your case, each statement just appears to be something that __could__ be said by a user, the issue is that it doesn't provide ChatterBot with any information about how to respond to these inputs.